synfig-studio/plugins/lottie-exporter/common/Param.py

```diff
@@ -518,6 +518,18 @@ def recur_animate(self, anim_type):
                 self.expression = ret
                 return ret, self.expression_controllers
 
+            elif self.param[0].tag == "atan2":
+                self.subparams["atan2"].extract_subparams()
+                y, eff_1 = self.subparams["atan2"].subparams["y"].recur_animate(anim_type)
+                x, eff_2 = self.subparams["atan2"].subparams["x"].recur_animate(anim_type)
+                self.expression_controllers.extend(eff_1)
+                self.expression_controllers.extend(eff_2)
+                
+                ret = "radiansToDegrees(Math.atan2({y}, {x}))"
```

Comment:
This expression is not giving the correct angle. You might need to add/subtract 180 degrees depending upon the axis used by lottie. Please try this expression on some more examples.

Reply:
Added a fix for that mistake, works fine depending on the axis now ^_^

Reply:
Yep, works perfectly now!
Nice work.